osdep/threads.h

```diff
@@ -81,6 +83,10 @@ enum mp_mutex_type {
 #define mp_mutex_init_type(mutex, mtype) \
     assert(!mp_mutex_init_type_internal(mutex, mtype))
 
-#include "threads-posix.h"
+#if HAVE_WIN32_THREADS
+# include "threads-win32.h"
+#else
+# include "threads-posix.h"
```

Comment:
```suggestion
#include "threads-win32.h"
#else
#include "threads-posix.h"
```

More nits while I'm at it.

Reply:
done